crud/crud.cpp

```diff
@@ -54,6 +54,12 @@ crud::start()
                 {
                     bzn_msg msg;
 
+                    // TODO: This is a hack to stop a crash
```

Comment:
Your comment is misleading. It should read:

This is to avoid trying to process encoded protobuf messages where we just want to commit this index, but not alter storage at all.

Reply:
Cool, making the change.